src/components/Alert/__snapshots__/Alert.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Vitest Snapshot v1
+
+exports[`<Alert> > renders and matches snapshot 1`] = `<div />`;
```

Comment:
Am I missing something here? Why is this just an empty div?

Reply:
I mocked the outer container to get the correct snapshot.